torch/quantization/_numeric_suite.py

```diff
@@ -3,6 +3,41 @@
 import torch
 import torch.nn as nn
 import torch.nn.quantized as nnq
+from torch.quantization import (
+    DEFAULT_DYNAMIC_MODULE_MAPPING,
+    DEFAULT_MODULE_MAPPING,
+    DEFAULT_QAT_MODULE_MAPPING,
+    DeQuantStub,
+    RecordingObserver,
+    prepare,
+)
+
+
+_EXCLUDE_QCONFIG_PROPAGATE_LIST = {DeQuantStub}
+_INCLUDE_QCONFIG_PROPAGATE_LIST = {nn.Sequential}
+
+
+def compute_error(x, y, error_type="l2"):
```

Comment:
Is this function used in the code?

Reply:
It's not used in the code, but for user to use when they get the dict from compare APIs to compute the error.

Reply:
Please remove it as it is close to torch.norm, we can add it if needed later.